cpp/modules/luma.gl/core/src/model.cpp

```diff
@@ -41,7 +41,12 @@ Model::Model(wgpu::Device device, const Model::Options& options) {
   descriptor.vertexStage.module = this->vsModule;
   descriptor.cFragmentStage.module = this->fsModule;
   descriptor.primitiveTopology = options.primitiveTopology;
+
   descriptor.cColorStates[0].format = options.textureFormat;
+  descriptor.cColorStates[0].colorBlend.srcFactor = wgpu::BlendFactor::SrcAlpha;
```

Comment:
Nice to see blending coming into place.

Does this mean that in WebGPU the entire model has to be recreated just to change such parameters?

The JS API allows for a prop called `parameters` which is an object taking all the GL parameters like this (i.e. this is exposed to the JS and JSON APIs).

Such objects can be passed in to draw calls and be changed between e.g. rendering and picking (picking must always be done without blending as the pixel values of colors are the key to deducing what was picked).

Worst case we need separate models for picking and drawing. We could either hide this inside the Model class or make the layer create two Models

Reply:
I went through the entire API on https://gpuweb.github.io/gpuweb as it's the only reference I have, and blending is only mentioned as part of this API, and [setBlendColor](https://gpuweb.github.io/gpuweb/#dom-gpurenderpassencoder-setblendcolor), which I assume isn't what we're looking for?

For our current feature set, is hardcoding these sufficient? Should alpha blending also use alpha/1-alpha factors?

Reply:
We'll have to change this at some point, but this is good enough for the purposes of the initial porting project.

Reply:
Updated `alphaBlend.srcFactor/dstFactor`, are these the factors that make sense for the time being?